docker/dist-build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+mkdir -p dist
+
+bazel build //:libtrtorch //:bin --compilation_mode opt
```

Comment:
Were you not finding trtorchc in the tarball?

Reply:
Oh you mean //:libtrtorch includes //bin ? I will clean this up though it's probably a no-op